Remove forward slash from URL regex path part (#25216)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_11_48.md

#### Scripts

##### FormatURL

> - Fixed an issue that caused a function to into an infinite loop.

Reviewer: ```suggestion
- Fixed an issue that caused the script to enter into an infinite loop.
```
